corehq/apps/users/cases.py

```diff
@@ -28,6 +28,13 @@ def _get_class(doc_type):
             'Group': Group,
         }.get(doc_type)
 
+    def _get_deleted_class(doc_type):
+        return {
+            'CommCareUser-Deleted': CommCareUser,
+            'WebUser-Deleted': WebUser,
```

Comment:
I think this is actually `CouchUser-Deleted`: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/models.py#L1675